### DISCUSSION TITLE
Difficulties creating the config.json file

### DISCUSSION THREAD

**Discussion by Raulabc** [lukearran/TvTimeToTrakt#23] [Q&A] [answered]
I'm inexperienced in these computer issues. I just wanted to transfer my TvTime history to Trakt, but I do all the steps and when I run TimeToTrakt.py, it tells me that the config.json was not found. Someone can help me? I created the aforementioned file with notepad, adding the .json ending.
Thanks in advance.

**Comment by myvanity** [comment, marked as answer]
I saved the config file in the same directory as TimeToTrakt.py and opened the terminal in said directory.
